apps/ledger-live-mobile/src/components/CopyLink.tsx

```diff
@@ -0,0 +1,90 @@
+// @flow
```

Comment:
Do we need to transform the file using désign system and style component or is it ok like this?